### DISCUSSION TITLE
Facing issues while building Orbit

### DISCUSSION THREAD

**Discussion by juhyung-k** [google/orbit#4733] [Q&A]
Hi,

I'm trying to build the project on CentOS 7 to use the Orbit, but I am encountering issues.

1. When building from the current_release branch (1.85), the build fails with the following error message.
```
Profile created with detected settings: /home/joshua/.conan/profiles/default
abseil/20211102.0: Not found in local cache, looking in remotes...
abseil/20211102.0: Trying with 'bintray'...
...
...
...
Response from remote is not json, but 'text/html; charset=utf-8'. [Remote: bintray]
```

It appears that I can access the conan-center (https://center.conan.io/), but not bintray (https://orbit.jfrog.io/artifactory/api/conan/orbitdeps).

2. When building from the main branch, the build was successful but some tests failed and the project doesn't seem to work properly. The specific test failures include:

```
448 - AllocateInTraceeTest.AllocateAndFree (Failed)
8995     2665 - HttpDownloadManagerTest.DownloadSingleSucceeded (Timeout)
8996     2666 - HttpDownloadManagerTest.DownloadSingleCanceled (Timeout)
8997     2667 - HttpDownloadManagerTest.DownloadSingleInvalidUrl (Timeout)
8998     2668 - HttpDownloadManagerTest.DownloadSingleInvalidSaveFilePath (Timeout)
8999     2669 - HttpDownloadManagerTest.DownloadMultipleSucceeded (Timeout)
9000     2670 - HttpDownloadManagerTest.DownloadSingleDestroyManagerEarly (Timeout)
9001     2921 - MicrosoftSymbolServerSymbolProviderIntegrationTest.RetrieveWindowsPdbAndLoadDebugSymbols (Timeout)
```

What can I do to build and use this project? If there's anything I'm missing, please let me know. I would appreciate it.

**Comment by florian-kuebler** [comment]
Hey, sorry to hear that you ran into build issues.

So the release branches are all from times prior to our build system overhaul, so they are not really supported anymore. Building main is the best way to do, right now.

The failing tests might be a permission issue, at least the build is done at that point (which is good). If you post the details for the failing test, we might be able to figure out more.

>the build was successful but some tests failed and the project doesn't seem to work properly

What are the "symptoms" that you are seeing (despite the failing tests) that let you assume it does not work? What exactly are you trying to do and what is failing?

I hope we can figure out the problem.
